src/html/index.html

```diff
@@ -12,7 +12,7 @@
   {% include "partials/problem.html"%}
   {% include "partials/key-factors.html"%}
   {% include "partials/question.html"%}
-  {% include "partials/concept.html"%}
+  <!-- {% include "partials/concept.html"%} -->
```

Comment:
Is there a way to comment this out using the templating language? That way it doesn't show up in the markup.

Reply:
That's temporary, the images in that partial were messing with the overall width of the page so I disabled until I'm ready to work with it.